graph-db/package.json

```diff
@@ -38,5 +38,10 @@
     "nock": "^13.1.1",
     "serverless-offline": "^6.9.0",
     "serverless-pseudo-parameters": "^2.5.0"
+  },
+  "resolutions": {
+    "adm-zip": "^0.5.2",
```

Comment:
With the serverless package updated to the latest version, we don't need this anymore.

Reply:
by this, I mean the whole resolutions block.